core/eolearn/core/utils/fs.py

```diff
@@ -118,6 +126,58 @@ def get_aws_credentials(aws_profile: str, config: Optional[SHConfig] = None) ->
     return config
 
 
+def pickle_fs(filesystem: FS) -> bytes:
+    """By default, filesystem objects cannot be pickled because they contain thread locks and similar unserializable
+    objects. This function removes the before pickling the filesystem object.
```

Comment:
Probably something wrong in the sentence - weird grammar.

Reply:
Thanks, it should be better now.